index.html

```diff
@@ -4124,6 +4135,60 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
         </section>
       </section>
 
+      <section id="dWLm-chunk">
+        <h2><span class="chunk">dWLm</span> Diffuse white luminance metadata</h2>
+        <p>The two-byte chunk type field contains the hexadecimal values</p>
+
+        <pre>
+<!-- xx xx xx xx
+</pre>
+
+        <p>If present, the <span class="chunk">dWLm</span> chunk stores the luminance value 
+          of the images nominal diffuse white, in cd/m2 as defined in [ISO/TS 22028-5]:</p>
+
+        <p>If the luminance of the diffuse white differs from the nominal luminance for an HDR reference displays
+          diffuse white as defined in in [[ITU-R-BT.2408]], the contents diffuse white luminance level
+          should be indicated using this metadata.</p>
+           
+         <p>The following specifies the syntax of the <span class="chunk">dWLn</span> chunk:</p>
+
+         <table id="dWLn-chunk-syntax" class="numbered simple">
+           <caption>dWLn chunk components</caption>
+
+           <tr>
+             <th>Name</th>
+             <th>Size</th>
+             <th>Divisor value</th>
+           </tr>
+
+           <tr>
+             <td>Diffuse white luminance metadata</td>
+             <td>2 bytes</td>
```

Comment:
Here's the HEIF TuC document that contains proposed language for this same piece of metadata (look for `ndwt`):
https://www.mpeg.org/wp-content/uploads/mpeg_meetings/144_Hannover/w23211.zip

I think it would be beneficial to make this a `uint32_t` with a unit of 0.0001 cd/m^2 so that we could have lossless metadata conversion between PNG and HEIF.

I have pinged folks internally on whether `diffuse_white_light_x/y` would be good additions as well.

Reply:
@podborski mentioned the following regarding the x/y values:
> IIRC it was bound to the reference viewing environment and was specified in 22028-5 and we needed a way to carry that metadata in HEIF.

Reply:
I'm not sure I follow. And I don't have access to ISO 22028-5 to check for myself.
Does 22028-5 specify 4 bytes here? Thus, this comment would put us in line with both 22028-5 and HEIF?
Or does 22028-5 specify 2 bytes and HEIF is different from 22028-5?

Reply:
> I'm not sure I follow. And I don't have access to ISO 22028-5 to check for myself. Does 22028-5 specify 4 bytes here? Thus, this comment would put us in line with both 22028-5 and HEIF? Or does 22028-5 specify 2 bytes and HEIF is different from 22028-5?

From ISO/DTS 22028-5:

> **4.5.5 Default nominal diffuse white luminance**

> The default nominal diffuse white luminance shall be 203 cd/m2.

> NOTE This nominal luminance value is defined in ITU-R BT.2408-4 as the HDR Reference White.

> If the reference nominal diffuse white for specific content has a different displayed luminance, the
reference nominal diffuse white for that content should be indicated using metadata, as defined in 4.6.5.

and then

> **4.6.5 Diffuse white luminance metadata**

> The diffuse white luminance metadata stores the luminance value of the reference nominal diffuse
white, in cd/m2.

> If the luminance of the diffuse white differs from the default reference display nominal diffuse white
luminance defined in 4.5.5, it should be indicated using metadata.

So the encoding is not defined at all. Also, it is clear that this metadata is only to be used when media white has a luminance _other than_ 203 cd/m<sup>2</sup>

Reply:
The `diffuse_white_light_x/y` fields have been removed from the HEIF text. Latest text can be found here:
https://dms.mpeg.expert/doc_end_user/documents/147_Sapporo/wg11/MDS24143_WG03_N01297.zip

Reply:
That looks like a good change to me. Merely noting a different white point without also specifying a chromatic adaptation transform (CAT) is not especially helpful, and the most useful information (for PQ, at least) is the absolute luminance of the media white.